File: app/src/main/java/com/damianmichalak/shopping_list/model/api_models/ShoppingList.java
package com.damianmichalak.shopping_list.model.api_models;

import com.google.firebase.database.PropertyName;

import javax.annotation.Nonnull;

public class ShoppingList {

    @Nonnull
    @PropertyName("date_created")
    private long dateCreated;
    @Nonnull
    private String name;
    @Nonnull
    private String id;

    @Nonnull
    public long getDateCreated() {
        return dateCreated;
    }

    public String getName() {
        return name;
    }

}

File: app/src/main/java/com/damianmichalak/shopping_list/model/apiModels/ShoppingList.kt
package com.damianmichalak.shopping_list.model.apiModels

import com.google.firebase.database.PropertyName

data class ShoppingList(@PropertyName("date_created")
                        val dateCreated: Long = 0,
                        val name: String = "",
                        private val id: String = "")
